### DISCUSSION TITLE
Terraform Cloud support

### DISCUSSION THREAD

**Discussion by estahn** [suzuki-shunsuke/tfcmt#811] [Deprecated (Question and Answer)]
We're using Terraform Cloud. A commit triggers a Terraform cloud run and reports the result (see below) but no details. The details need to be taken by going to Terraform Cloud. It would be ideal to have this all on the PR. Any idea if this could be supported?

![image](https://github.com/suzuki-shunsuke/tfcmt/assets/362174/d23896f6-affa-4fac-9381-8e6557b31ef3)

**Comment by suzuki-shunsuke** [comment]
I'm not familiar with Terraform Cloud, so I'm not sure if Terraform Cloud supports running tools such as tfcmt.
How about asking Terraform Community at https://discuss.hashicorp.com/c/terraform-core/27 or somewhere?
